crates/electrum/tests/test_electrum.rs

```diff
@@ -101,7 +101,8 @@ fn scan_detects_confirmed_tx() -> anyhow::Result<()> {
             .fee
             .expect("Fee must exist")
             .abs()
-            .to_sat() as u64;
+            .to_unsigned()
+            .expect("valid `SignedAmount");
```

Comment:
missing backtick, or remove the dangling one.

Reply:
done at [a03949a](https://github.com/bitcoindevkit/bdk/pull/1426/commits/a03949adb0c38ba4c2f44b497871070c1a2170d2)

Reply:
but you called `to_unsigned` so the expect message is wrong. oh well

Reply:
> but you called `to_unsigned` so the expected message is wrong. oh well

Oh, sorry for the oversight, I fixed it in #1458 🤦‍♂️